crates/store/src/db/tests.rs

```diff
@@ -1601,18 +1609,17 @@ fn genesis_with_multiple_accounts() {
 
     let component_storage = vec![StorageSlot::with_map(StorageSlotName::mock(0), storage_map)];
 
-    let component3 = AccountComponent::compile(
-        "export.baz push.3 end",
-        TransactionKernel::assembler(),
-        component_storage,
-    )
-    .unwrap()
-    .with_supported_type(AccountType::RegularAccountImmutableCode);
+    let account_component_code = CodeBuilder::default()
+        .compile_component_code("baz::interface", "export.baz push.3 end")
```

Comment:
FYI: After https://github.com/0xMiden/miden-base/pull/2158 is merged, all `export`s will have to become `pub proc`.